packages/flutter/lib/src/rendering/sliver_list.dart

```diff
@@ -101,14 +101,22 @@ class RenderSliverList extends RenderSliverMultiBoxAdaptor {
     // offset.
     if (childScrollOffset(firstChild!) == null) {
       int leadingChildrenWithoutLayoutOffset = 0;
-      while (childScrollOffset(earliestUsefulChild!) == null) {
+      while (earliestUsefulChild != null && childScrollOffset(earliestUsefulChild) == null) {
         earliestUsefulChild = childAfter(firstChild!);
```

Comment:
Now that I look at it, shouldnt this also be `childAfter(earliestUsefulChild)`?

Reply:
@Kavantix Yes, I think so

Reply:
oops nice catch